src/casm/crystallography/Site.cc

```diff
@@ -88,8 +88,7 @@ SiteDoFSet const &Site::dof(std::string const &_dof_type) const {
 
 //****************************************************
 bool Site::has_dof(std::string const &_dof_type) const {
-  return occupant_dof().size() > 1 ||
-         m_dof_map.find(_dof_type) != m_dof_map.end();
+  return m_dof_map.find(_dof_type) != m_dof_map.end();
```

Comment:
```suggestion
  return    (_dof_type == "occ" && occupant_dof().size() > 1)
         || m_dof_map.find(_dof_type) != m_dof_map.cend();
```
I suspect you'll get in trouble if you exclude the occupation case. But, I'd also recommend not using the raw string, `"occ"` (and defining it somewhere globally instead).